src/repository/msgraph/participant.repository.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
```

Comment:
nu cred ca ai nevoie de repo si entity daca nu faci nici o operatie in DB :)

Reply:
am sters cele 2 foldere de ms grpah entity si ms graph repo si mutat fisierele la locul lor